docs/book/src/developer/tilt.md

```diff
@@ -186,8 +186,18 @@ tilt up
 
 This will open the command-line HUD as well as a web browser interface. You can monitor Tilt's status in either
 location. After a brief amount of time, you should have a running development environment, and you should now be able to
-create a cluster. Please see the [Usage section in the Quick
-Start](https://cluster-api.sigs.k8s.io/user/quick-start.html#usage) for more information on creating workload clusters.
+create a cluster. There are [example worker cluster
+configs](https://github.com/kubernetes-sigs/cluster-api/tree/master/test/infrastructure/docker/examples) available.
+These can be customized for your specific needs.
+
+<aside class="note">
+
+<h1>Use of clusterctl</h1>
+
+When the worker cluster has been created using tilt, it is not recommended to use `clusterctl` for management
```

Comment:
I would add why here

Reply:
I can add something, but honestly I'm not sure if I fully understand why. I just went with this based on Fabrizio's statement that clusterctl should not be used if the cluster was not initialized with it.

Do you have a little more context on that @CecileRobertMichon? Happy to add more details.

Reply:
the TLDR is that tilt doesn't initialize providers on the management cluster like `clusterctl init` does, so some of the `clusterctl` commands like `clusterctl config` won't work. @fabriziopandini might be able to provide a more detailed answer.

Reply:
I took a crack at it, but please let me know if I'm too far off base. ;)

Reply:
Shoot, I should have refreshed sooner. Will hold for more feedback before making any more changes.